runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaCollectorTask.kt

```diff
@@ -1,59 +1,29 @@
 package org.jetbrains.dokka.gradle
 
-import com.google.gson.GsonBuilder
-import org.gradle.api.plugins.JavaBasePlugin.DOCUMENTATION_GROUP
-import org.gradle.api.tasks.Input
-import org.jetbrains.dokka.toJsonString
+import org.jetbrains.dokka.DokkaConfigurationImpl
 
-open class DokkaCollectorTask : AbstractDokkaTask() {
+open class DokkaCollectorTask : AbstractDokkaParentTask() {
```

Comment:
Probably just 💄 , but I would recommend making all task classes abstract instead of open.

Reply:
Thanks for the hint! ☺️ 
Pushed a commit to change this ✅